### PR TITLE
#6335: Sorting fix for attributes table

### PR DESCRIPTION
**Pull request by dsuren1**
## Description
This PR adds fix for sorting issue in attribute table 

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Bugfix

## Issue

**What is the current behavior?**
#6335

**What is the new behavior?**
Attributes columns are sorted based on type of sorting applied to the column in attribute table 

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: web/client/epics/__tests__/wfsquery-test.js [hunk 1]
```diff
@@ -94,6 +94,38 @@ describe('wfsquery Epics', () => {
             }
         });
     });
+    it('wfsQueryEpic passes filterObj', (done) => {
```
Comment (MV88) [inline web/client/epics/__tests__/wfsquery-test.js:97]: i would use a more meaningful test title here, aren't you testing that is using default sorting if not provided one?

**Review by MV88 — CHANGES_REQUESTED** [review]
hi, @dsuren1 all good, just maybe the test title 

also maybe you can refactor a bit how mock axios is used to instantiate it ina single point 
 **before each** and to restore in the **after each**

similar to [this](https://github.com/geosolutions-it/MapStore2/blob/master/web/client/actions/__tests__/maps-test.js#L47-L57)
without the geostore part

File: web/client/epics/__tests__/wfsquery-test.js (hunk 1 above)
Comment (dsuren1) [reply web/client/epics/__tests__/wfsquery-test.js:97]: Yes. I will modify the titles accordingly

**Review by MV88 — APPROVED** [review]

**Comment by MV88** [comment]
please @dsuren1 backport this to 2021.01.xx

**Pull request merged**